### ISSUE TITLE
Plurals don't show if no translation found.

### ISSUE THREAD

**Issue by hworld** [Polyconseil/vue-gettext#16]
```
<translate
	:translate-n="count"
	translate-plural="%{ count } items."
	>
	%{ count } item.
</translate>
```

If you don't have the translation yet for the string in the library, then it will always fall back to the singular version. This makes testing in dev a bit tricky. Right now in the getTranslation function it returns the msgid if it can't find the string. I think it could be possible to check the value of n, use that with getTranslationIndex, and return from the `translate-plural` param when there is no translation yet.

Might be a bit hacky, but even in production I don't always have translations for languages yet, and it would be great to just fallback in those scenarios. 

Thoughts?

**Comment by kemar** [comment]
@hworld sorry I don't understand your problem here. Do you have a full example that I can reproduce?

**Comment by hworld** [comment]
@kemar Sure thing. Here's a fiddle: https://jsfiddle.net/Ld9rh997/

You can see it shows the same translation string for both 1 and 10 count. It's because it's earlying out and returning msgid directly instead of pulling in the translate-plural message ID.

**Comment by hworld** [comment]
Had the wrong fiddle before. Just updated it.

**Comment by kemar** [comment]
@hworld ok thanx, it's crystal clear now!

I could hack around and return the `translate-plural` param but that would be really hacky.

Here is why: some languages have [multiple plural forms](http://docs.translatehouse.org/projects/localization-guide/en/latest/l10n/pluralforms.html).

E.g.: Arabic has 6 plural forms which could be in the translation file but not in the component's HTML. It's impossible to fallback to the correct plural form in this case. So returning `translate-plural` may work for your language, but will still fail for others.

Translating requires discipline and developers probably shouldn't deploy an untranslated application.

**Comment by hworld** [comment]
@kemar I'm aware of the multiple plural forms issue, however this isn't how gettext deals with it. The idea of a gettext enabled program is that it has defaults that it shows if untranslated. This allows projects to still function and be translated by volunteers as well as can be expected. It's very common in the open source world to have programs that aren't fully translated in all languages supported.

It's supposed to fall back to using Germanic language rules when there is no translation for the msgid. One singular, and one plural form. Otherwise, why would you even have the plural form in the code?  You could just as easily mark "this should be pluralized" and then they can use their chosen PO file editor to add the plural versions.

You read the documentation for gettext on plural forms here: https://www.gnu.org/software/gettext/manual/gettext.html#Plural-forms

I'll copy out the snippet that is related to this discussion:

> This has the consequence that programs without language catalogs can display the correct strings only if the program itself is written using a Germanic language. This is a limitation but since the GNU C library (as well as the GNU gettext package) are written as part of the GNU package and the coding standards for the GNU project require program being written in English, this solution nevertheless fulfills its purpose.

gettext assumes English (or at least a Germanic language) as the source. Otherwise we'd be instead marking messages with actual message IDs (`dashboard.overview.sale_count`) and translating them from that.

Hope this helps describe my reasoning for it. I think it's in following the ideals of how gettext is supposed to work.

**Comment by kemar** [comment]
@hworld ok that makes sense.

I'll read this part of the documentation and I'll make a decision based on it.

**Comment by kemar** [comment]
@hworld thanx for your detailed feedback.

Since it totally makes sense I just released v2.0.10 which adds this enhancement.

**Comment by hworld** [comment]
Whoa, that was really fast! Thanks a lot!
